changelog.md

```diff
@@ -15,7 +15,10 @@
      `type role MVector nominal representational` (previously, both arguments
      were `phantom`).
    * The role signature for `Data.Vector.Primitive.Vector` is now
-     `type role Vector representational` (previously, it was `phantom`).
+     `type role Vector nominal` (previously, it was `phantom`).
+     for `Data.Vector.Primitive.Mutable.MVector` is now
+     `type role MVector nominal nominal` (previous, both arguments were
+     `phantom`).
```

Comment:
"for" -> "For" (or preferably put "The role signature for" before) 
"previous" -> "previously".